src/main/java/unibook/logic/commands/EditCommand.java

```diff
@@ -191,10 +232,11 @@ public CommandResult execute(Model model, Boolean isPersonListShowing,
 
             // Find all profs and students with this module and will be edited
             for (Person person : lastPersonList) {
-                Set<Module> moduleSet = person.getModules();
+                Set<Module> moduleSet = person.getModulesModifiable();
                 if (moduleSet.contains(moduleToEdit)) {
                     moduleSet.remove(moduleToEdit);
                     moduleSet.add(editedModule);
+                    person.setModules(moduleSet);
```

Comment:
I feel that this line
`person.setModules(moduleSet)`
is redundant actually, as the person's moduleSet is already beinng modified in the lines above. If my understanding is correct, the person's moduleSet will still be updated without needing this line. What do you think? 🤔